<launch>
    <group ns="tf03_driver">
        <rosparam param="sensor_interface">can</rosparam>
        <rosparam param="can_transmit_id"> [1]</rosparam>
        <rosparam param="can_receive_id"> 17</rosparam>
        <rosparam param="sensor_frame"> ["front"]</rosparam>        
        <rosparam param="can_device">benewake_can</rosparam>
    </group>
    <node pkg="tf03_driver" name="tf03_driver" type="tf03_communication_node" output="screen" required="true">
    </node>
</launch>
